cc/action_names.bzl

```diff
@@ -162,12 +164,14 @@ ALL_CC_LINK_ACTION_NAMES = [
     ACTION_NAMES.lto_index_for_executable,
     ACTION_NAMES.lto_index_for_dynamic_library,
     ACTION_NAMES.lto_index_for_nodeps_dynamic_library,
+    ACTION_NAMES.objc_executable,
```

Comment:
I had to revert this line - breaks 1 tiny test. If we want to add it back, it'll need a bit longer investigation.